<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>141</width>
     <height>521</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>550</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>550</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_user">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="UserWidgetInfo" name="widget1" native="true">
       <property name="minimumSize">
        <size>
         <width>778</width>
         <height>523</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>778</width>
         <height>523</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_buyer">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="BuyerWidget" name="widget2" native="true">
       <property name="minimumSize">
        <size>
         <width>778</width>
         <height>523</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>778</width>
         <height>523</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_seller">
    <widget class="SellerWidget" name="widget3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>778</width>
       <height>523</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>778</width>
       <height>523</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>778</width>
       <height>523</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>511</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserWidgetInfo</class>
   <extends>QWidget</extends>
   <header location="global">userwidgetinfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SellerWidget</class>
   <extends>QWidget</extends>
   <header>sellerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BuyerWidget</class>
   <extends>QWidget</extends>
   <header>buyerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
